README.md

```diff
@@ -12,7 +12,7 @@ For the configurations' entry points see the individual [systems], as well as
 ```
 .
 ├── core         # Base-line configurations applicable to all machines
-├── default.nix  # Attr set of all machines plus arch-groups
+├── default.nix  # Attr set of all machines, with groups based on architecture
```

Comment:
Now that I think about this with a rested mind, it would look better and sound less awkward as:
```suggestion
├── default.nix  # Attr set of all machines, grouped by architecture
```
;)